/**:
  ros__parameters:
    board_node: "192.168.1.80:5554"
    pilot_config: "/home/neobotix/ros2_workspace/src/neo_relayboard_v3/config/default/rox-argo"
    agm_batteries_config: "/home/neobotix/ros2_workspace/src/neo_relayboard_v3/config/addons/agm-batteries"
    lfp_batteries: False # Set to True if you have LFP Batteries
    lfp_batteries_config: "/home/neobotix/ros2_workspace/src/neo_relayboard_v3/config/addons/lfp-batteries"
    use_multipowr: False #Set to True if you have wallbox
    use_multipowr_config: "/home/neobotix/ros2_workspace/src/neo_relayboard_v3/config/addons/smart-charger"
